Content.Client/Light/Visualizers/LightBulbVisualizerSystem.cs

```diff
@@ -0,0 +1,36 @@
+using Content.Shared.Light;
+using Robust.Client.GameObjects;
+
+namespace Content.Client.Light.Visualizers;
+
+public sealed class LightBulbVisualizerSystem : VisualizerSystem<LightBulbVisualizerComponent>
```

Comment:
merge into LightBulbSystem

Reply:
No such system existed in shared or client. Had to move LightBulbComponent to shared.